make_package.sh

```diff
@@ -12,18 +12,20 @@ if [ "$VERSION" == "" ]; then
   exit 1
 fi
 
+PACKAGE_DIR=package-$VERSION
+PROJECT_DIR_NAME=datastax-mcac-agent-$VERSION
+
 OPTIONAL_MVN_ARGS=""
 if [ -n "$CASSANDRA_VERSION" ]; then
   OPTIONAL_MVN_ARGS="${OPTIONAL_MVN_ARGS} -Dcassandra.version=${CASSANDRA_VERSION}"
+  # make a different package for the specific Cassandra version
+  PROJECT_DIR_NAME=datastax-mcac-agent-$VERSION-${CASSANDRA_VERSION}
```

Comment:
Calling this script with no Cassandra Version argument will build the bundles as usual, and they will work for versions of Cassandra 4.0.x and lower. The release action above will call this script a second time, specifying a Cassandra version, so we run it again, but create a different project directory to stage and bundle the version for Cassandra 4.1 and newer.